lib/tasks/cap_cutover.rake

```diff
@@ -478,199 +59,4 @@ namespace :cap_cutover do
-          @no_pubmed_rec << pub.id
-        end
-        return
-      end
-      @cap_import_logger.info("Pub: #{pub.id} fixing missing PubmedSourceRecord")
-      pubmed_hash = pubmed_source_record.source_as_hash
-      pub.update_attributes(
-        active: true,
-        title: pubmed_hash[:title],
-        year: pubmed_hash[:year],
-        pub_hash: pubmed_hash)
-      changed = true
-    end
-    if changed
-      pub.cutover_sync_hash_and_db
-      pub.save
-    end
-
-  rescue => e
-    @cap_import_logger.info 'Problem with pub: ' + pub.id.to_s
-    @cap_import_logger.info e.message << "\n" << e.backtrace.join("\n")
-  end
-
-  def self.convert_manual_publication_row_to_hash(cap_pub_data_for_this_pub, author_id)
-    record_as_hash = {}
-
-    record_as_hash[:provenance] = Settings.cap_provenance
-    record_as_hash[:title] = cap_pub_data_for_this_pub[:article_title]
-    # record_as_hash[:abstract_restricted] = cap_pub_data_for_this_pub[:abstract] unless cap_pub_data_for_this_pub[:abstract].blank?
-    primary_author = cap_pub_data_for_this_pub[:primary_author]
-    if primary_author.present?
-      record_as_hash[:author] = [{ name: primary_author }]
-    else
-      record_as_hash[:author] = []
-    end
-
-    unless cap_pub_data_for_this_pub[:authors].blank?
-      record_as_hash[:author].concat(cap_pub_data_for_this_pub[:authors].split(',').collect { |author| { name: author } })
-    end
-
-    record_as_hash[:year] = cap_pub_data_for_this_pub[:publication_date] unless cap_pub_data_for_this_pub[:publication_date].blank?
-
-    record_as_hash[:type] = Settings.sul_doc_types.article
-
-    record_as_hash[:country] = cap_pub_data_for_this_pub[:country] unless cap_pub_data_for_this_pub[:country].blank?
-
-    record_as_hash[:identifier] = [{ type: 'legacy_cap_pub_id', id: cap_pub_data_for_this_pub[:deprecated_publication_id] }]
-
-    journal_hash = {}
-    journal_hash[:name] = cap_pub_data_for_this_pub[:publication_title] unless cap_pub_data_for_this_pub[:publication_title].blank?
-    journal_hash[:volume] = cap_pub_data_for_this_pub[:volume] unless cap_pub_data_for_this_pub[:volume].blank?
-    journal_hash[:issue] = cap_pub_data_for_this_pub[:issue_no] unless cap_pub_data_for_this_pub[:issue_no].blank?
-    journal_hash[:pages] = cap_pub_data_for_this_pub[:page_ref] unless cap_pub_data_for_this_pub[:page_ref].blank?
-    issn = cap_pub_data_for_this_pub[:issn]
-    record_as_hash[:issn] = issn unless issn.blank?
-    journal_hash[:identifier] = [{ type: 'issn', id: issn }] unless issn.blank?
-    record_as_hash[:journal] = journal_hash unless journal_hash.empty?
-
-    record_as_hash[:authorship] = [
-      {
-        cap_profile_id: cap_pub_data_for_this_pub[:profile_id],
-        sul_author_id: author_id,
-        status: cap_pub_data_for_this_pub[:authorship_status],
-        visibility: cap_pub_data_for_this_pub[:visibility],
-        featured: cap_pub_data_for_this_pub[:featured]
-      }
-    ]
-
-    record_as_hash
-  end
```

Comment:
`convert_manual_publication_row_to_hash` is not in #356 analysis but does seem to be a private method called by rake tasks to be deleted.